spikeinterface/curation/auto_merge.py

```diff
@@ -0,0 +1,264 @@
+import numpy as np
+import scipy.signal
+import scipy.spatial
+
+from ..postprocessing import compute_correlograms, get_template_extremum_channel
+
+# TODO:
+#   * adaptative window p(aka plage) on CC
+#   * template similarity
+#   * 
+
+def get_potential_auto_merge(waveform_extractor,
+                minimum_spikes=1000, maximum_distance_um=200.,
+                peak_sign="neg",
+                bin_ms=0.25, window_ms=50., corr_thresh=0.3,
+                correlogram_low_pass = 800., adaptative_window_threshold=0.5,
+                debug_folder=None,
+                ):
+    """
+    Algorithm to find and check potential merges between units.
+    
+    This is taken from lussac version 1 done Aurelien Wyngaard.
+    https://github.com/BarbourLab/lussac/blob/main/postprocessing/merge_units.py
```

Comment:
Use this link (to point to v1.0)
https://github.com/BarbourLab/lussac/blob/v1.0.0/postprocessing/merge_units.py